lib/view/info_view.dart

```diff
@@ -75,41 +88,49 @@ class InfoView extends StatelessWidget {
         iconTheme: Styles.appBarIconTheme,
         title: Text(
           title,
-          style: Styles.appBarTextStyle,
+          style: Styles.textH5,
         ),
       ),
       body: Padding(
-        padding: const EdgeInsets.all(16.0),
+        padding: const EdgeInsets.symmetric(horizontal: 16.0),
         child: ListView(
           children: <Widget>[
             _buildSpacer(),
-            ...topCards,
-            _buildSpacer(),
+            ...cards,
+            _buildSpacer(height: 24),
             ..._buildAbout(),
             _buildSpacer(),
-            ...bottomCards,
-            _buildSpacer(),
           ],
         ),
       ),
     );
   }
 
-  Widget _buildSpacer({double height = 32}) => Container(height: height);
+  static Widget _buildSpacer({double height = 16}) => Container(height: height);
 
   List<Widget> _buildAbout() {
     return [
-      Image.asset(
-        'assets/images/icon/high_res_icon.png',
-        height: 84,
-        width: 84,
-      ),
+      Center(
+          child: Container(
+              decoration: BoxDecoration(
+                  color: const Color.fromARGB(255, 12, 16, 62),
+                  borderRadius: BorderRadius.circular(6)),
+              padding: const EdgeInsets.all(4),
+              child: Image.asset(
+                'assets/images/icon/high_res_icon.png',
+                height: 50,
```

Comment:
usually all the dimens are multiples of 4, eg previously was 84 ?

Reply:
This is the size in the mock.
https://app.zeplin.io/project/5e7597a68651056664fb6e53/screen/5e75996c5f1e2a6dd0200cc1
![image](https://user-images.githubusercontent.com/15306325/78617632-44fefb00-78bb-11ea-8b45-01b023608e66.png)